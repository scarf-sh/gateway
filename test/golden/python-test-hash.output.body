<!DOCTYPE html><html><body><h1>numpy</h1>
<a href="/simple/numpy/numpy-1.3.0.tar.gz#md5=68b329da9893e34099c7d8ad5cb9c940" data-gpg-sig="true">numpy-1.3.0.tar.gz</a><br>
<a href="/simple/numpy/numpy-1.4.0.tar.gz#md5=60b725f10c9c85c70d97880dfe8191b3" data-requires-python=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*">numpy-1.4.0.tar.gz</a><br>
</body></html>