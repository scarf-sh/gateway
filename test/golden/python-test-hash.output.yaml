captures: |-
  [ Just
      PythonCapture
        { pythonCapturePackage = Just "123"
        , pythonCaptureVersion = Nothing
        , pythonCaptureFileName = Nothing
        , pythonCaptureFileBackendURL = Nothing
        }
  ]
headers:
  Content-Length: '351'
  Content-Type: text/html
  Etag: '"poRjCQOfzSLEnMunwDeaf9iLsReZf61anTfTPU2nUEg="'
status: 200
